src/libDirectoryService/Coinbase.cpp

```diff
@@ -157,13 +231,18 @@ void DirectoryService::InitCoinbase() {
   Address genesisAccount(GENESIS_WALLETS[0]);
 
   uint256_t sig_count = 0;
-
+  uint64_t lookup_count = 0;
```

Comment:
`lookupId` in `struct PoWSolution` is `uint32_t`, whereas for the count we use `uint64_t` here